### ISSUE TITLE
Failed to reproduce the results on VOC 2012 dataset

### ISSUE THREAD

**Issue by Justin0111** [VainF/DeepLabV3Plus-Pytorch#34]
Hi, VainF. Thanks for sharing this nice repo, where the code has great readability and practicability. However, I failed to reproduce the results on the voc dataset.

I trained deeplabv3plus-resnet101 (os 16, provided pre-trained weights for ResNet101) on the VOC 2012_aug dataset with all the other default settings but only changed the gpu_id to '0,1,2,3' as I couldn't train the model on one 2080ti gpu with the batch size of 16. And I also applied the SyncBN: https://github.com/vacancy/Synchronized-BatchNorm-PyTorch to avoid the performance decrease caused by multi-gpu training. And the best miou is 0.7539. Then I asked my friend to help to train the model, he trained the model on a TITAN RTX gpu with no SyncBN, and his best miou is 0.7535. Therefore I think multi-gpu training is fine with SyncBN. 

Did you further apply multi-scale inference for the validation? Do I need to change some settings to achieve 0.783 on VOC 2012_aug dataset?

Looking forward to your reply and suggestions. Thank you again for your effort.

**Comment by seyeeet** [comment]
@Justin0111 did you figure the answer?

**Comment by Justin0111** [comment]
@seyeeet No, I still failed to reproduce the results and I found another implementation which can be reproduced well.
This is the link: https://github.com/YudeWang/deeplabv3plus-pytorch. Hope this helps.

**Comment by seyeeet** [comment]
@Justin0111 Thanks for sharing
